src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_FOR_DAY = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+
+  if (days >= LONG_TERM) {
+    return days * PRICE_FOR_DAY - LONG_TERM_DISCOUNT;
```

Comment:
You do `days * PRICE_FOR_DAY ` 3 times, create a variable, for example `currentPrice`